### ISSUE TITLE
Feature request: Set project directory on Android

### ISSUE THREAD

**Issue by locrianz** [Wavesonics/hammer-editor#235]
This was originally a comment in #127 but then I thought better of it and decided it would be better as a feature request.

I'd love to be able to set the project directory on Android like I can on desktop, so that I can sync files with Syncthing and not have to mess around with hosting a server.

This project looks great so far, but this one change would make it absolutely incredible for me.

**Comment by Wavesonics** [comment]
Ya this has been on my list for a while. I'll get to it soon hopefully!

**Comment by arielsbecker** [comment]
Came here looking for the same. (Already checked #127).

This is my idea for a good use case:

* Have this app installed on both my desktop and phone.
* I work mostly on my desktop, but when I really need inspiration, I like to grab my bike, go in the middle of nowhere, and start typing while I _touch grass_.
* So once I'm back home, I would like to automatically sync via Syncthing, or push the changes to my own git server.
* So finally I have all the changes on my desktop, and I can keep working from there.

**Comment by Wavesonics** [comment]
yup this should be supported in the next version! Hoping to get it out in about a week.

**Comment by monospacedmagic** [comment]
I just went through the git history and noticed that you did implement this in v1.8.0, but decided to remove the feature in v1.8.1. Can you elaborate a little bit on what made you decide to remove the feature for the time being? A related code comment mentions that you were considering that maybe a different directory picker could solve the problem. I wish I could help more. 😅

**Comment by Wavesonics** [comment]
Ya I got it working, but it requires a special permission that Google has to individually approve for release on Google Play. I went through the application process, and was denied.

Google has really locked down app access to the public filesystem on Android in recent versions of Android.

So It looks like I won't be able to provide this feature on Google Play builds of the app. It should still be possible for side loading and FDroid builds, but I will need to do some work to actually produce different builds for GooglePlay VS the other distribution channels out there.

**Comment by FibroJedi** [comment]
Hey,

I am on the F-Droid installation and was trying to find the document location to be able to enable syncing between devices via a 3rd party system (like syncthing). This is the last step I need to be able to move story writing from Obsidian, which I want to do. 

I eventually found the app's folder but it has an access denied error (probably to prevent tampering with the actual app?) in the Android/data folder. But now I'm a bit stuck.

Can anyone let me know if they've found it, please? It's hard to sync two folders without knowing which folder on my phone to sync. Thanks!

**Comment by Wavesonics** [comment]
This is implemented in f-droid builds now

**Comment by FibroJedi** [comment]
Great, thank you. Time for an update!

-------- Original Message --------
